PubMatic/src/main/java/com/applovin/mediation/adapters/PubMaticMediationAdapter.java

```diff
@@ -161,6 +175,12 @@ public void onDestroy()
             adView.destroy();
             adView = null;
         }
+
+        if( nativeAd != null )
```

Comment:
NIT: Please correct the spacing between if and opening bracket to keep the consistency.
```suggestion
        if ( nativeAd != null )
```